<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-10-23T16:37:42 IST" name="Master Suite" finished-at="2024-10-23T16:37:59 IST" duration-ms="16754">
    <groups>
      <group name="Master">
        <method signature="TC005_AddToCartTest.verify_AddToCart()[pri:0, instance:testCases.TC005_AddToCartTest@242aa8d9]" name="verify_AddToCart" class="testCases.TC005_AddToCartTest"/>
      </group> <!-- Master -->
      <group name="Sanity">
        <method signature="TC005_AddToCartTest.verify_AddToCart()[pri:0, instance:testCases.TC005_AddToCartTest@242aa8d9]" name="verify_AddToCart" class="testCases.TC005_AddToCartTest"/>
      </group> <!-- Sanity -->
    </groups>
    <test started-at="2024-10-23T16:37:42 IST" name="Test" finished-at="2024-10-23T16:37:59 IST" duration-ms="16754">
      <class name="testCases.TC005_AddToCartTest">
        <test-method is-config="true" signature="setup(java.lang.String,java.lang.String)[pri:0, instance:testCases.TC005_AddToCartTest@242aa8d9]" started-at="2024-10-23T16:37:43 IST" name="setup" finished-at="2024-10-23T16:37:51 IST" duration-ms="8767" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Windows]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verify_AddToCart()[pri:0, instance:testCases.TC005_AddToCartTest@242aa8d9]" started-at="2024-10-23T16:37:51 IST" name="verify_AddToCart" finished-at="2024-10-23T16:37:58 IST" duration-ms="6667" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at testCases.TC005_AddToCartTest.verify_AddToCart(TC005_AddToCartTest.java:34)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_AddToCart -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.TC005_AddToCartTest@242aa8d9]" started-at="2024-10-23T16:37:59 IST" name="tearDown" finished-at="2024-10-23T16:37:59 IST" duration-ms="511" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.TC005_AddToCartTest -->
    </test> <!-- Test -->
  </suite> <!-- Master Suite -->
</testng-results>
